### PR TITLE
VIMTUTOR in Bulgarian - 100%

### PR DESCRIPTION
**Pull request by kberov**
Only in utf-8 encoding.
Thank you!

### CODE REVIEW

File: runtime/tutor/tutor.bg.utf-8 [hunk 1]
```diff
@@ -0,0 +1,1032 @@
...
+Забележете! Ако излезете от Vim и го пуснете отново като напишете на командния
+            ред  vim TEST , файлът ще бъде точно копие на самоучителя, когато
+            сте го записали.
+
+  5.  Сега изтрийте файла като напишете (в MS-DOS):    :!del TEST
+				или (в какъвто и да е Unix) :!rm TEST
+
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+		    Lesson 5.3: ИЗБОР НА ТЕКСТ ЗА ЗАПИС
```
Comment (dpelle) [inline runtime/tutor/tutor.bg.utf-8:689]: Is there any reason for not translating the word "Lesson"?

File: runtime/tutor/tutor.bg.utf-8 [hunk 2]
```diff
@@ -0,0 +1,1032 @@
...
+  5. Vim ще запише избраните редове във файла TEST.  Използвайте  :!dir  или  :!ls ,
+     за да го видите.  Не го изтривайте все още! Ще го използваме в следващия урок.
+
+Забележете! Като натиснете  v  започвате видимо избиране (Visual selection).  
+            Може да движите показалеца наоколо, за да направите избраното
+            по-голямо или по-малко. След което, можете да използвате оператор,
+            за да направите нещо с текста. Например,  d  изтрива текста.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+		   Lesson 5.4: ИЗВЛИЧАНЕ И СЛИВАНЕ НА ФАЙЛОВЕ
```
Comment (dpelle) [inline runtime/tutor/tutor.bg.utf-8:713]: Same remark: the word lesson is not translated.

File: runtime/tutor/tutor.bg.utf-8 [hunk 3]
```diff
@@ -0,0 +1,1032 @@
...
+  преглед на текстовия редактор Vim. Съвсем достатъчно, за да можете да
+  ползвате редактора лесно. Самоучителят е доста непълен, понеже Vim има много
+  повече команди. Сега прочете наръчника за потребителя: ":help user-manual".
+
+  Препоръчваме следната книга за по-нататъшно четене:
+	Vim - Vi Improved - от Steve Oualline
+	Издател: New Riders
+  Това е първата книга, изцяло посветена на Vim. Особено полезна е за
+  начинаещи. В нея ще намерите много примери и картинки.
+  See http://iccf-holland.org/click5.html
```
Comment (dpelle) [inline runtime/tutor/tutor.bg.utf-8:1015]: The word "See" is not translated.

**Comment by kberov** [comment]
Thank you @dpelle for spotting these. I will fix them now.

**Comment by brammool** [comment]
I'll include it, thanks.

**Comment by kberov** [comment]
I am fixing some typos, reviewing the whole thing once again. And wIll make another push. thanks.

**Pull request closed without merging**